dotnet/ibmmq_samples.sln

```diff
@@ -0,0 +1,30 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29020.237
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{E46345C8-4515-4845-B050-D9F0B37423DB}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ibmmq_samples", "ibmmq_samples\ibmmq_samples.csproj", "{6C890B5D-4826-4F5C-B929-414FE829946F}"

```

Comment:
I tried opening the solution but Visual Studio says that the project path is invalid. I think you need to change the path in this line like this:

```diff
- Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ibmmq_samples", "ibmmq_samples\ibmmq_samples.csproj", "{6C890B5D-4826-4F5C-B929-414FE829946F}"
+ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ibmmq_samples", "ibmmq_samples.csproj", "{6C890B5D-4826-4F5C-B929-414FE829946F}"
```